assets/chat/js/hashlinkconverter.js

```diff
@@ -24,6 +24,7 @@ class HashLinkConverter {
     const pathname = url.pathname.slice(1);
     let match;
     let videoId;
+    let timeStamp;
```

Comment:
A little nitpicky, but I'd prefer `timestamp` here over `timeStamp` for consistency.

Reply:
will sort